content/anoncomms/furps/incentivisation.md

```diff
@@ -0,0 +1,23 @@
+# Service Incentivisation FURPS
+
+## Functionality
+
+1. Client nodes can pay service nodes for decentralised services
+2. The payment protocol is mostly off-chain (no on-chain transaction necessary per service request)
+3. The payment protocol protects the privacy of the client by unlinking the payment from the service provision
+
+## Usability
+
+1. The payment protocol is published in a specification
+
+## Reliability
+
+1. 
+
+## Performance
+
+1. 
```

Comment:
Suggestion: off-chain protocol participation does not involve complex liquidity management, or locking up significantly more funds than required for the planned future payments.

Admittedly, this is quite vague, but what I'm trying to capture here is that, in Lightning lingo, we shouldn't make users have to deal with "incoming liquidity issue" or "routing issues".